src/components/staking/Undelegations.vue

```diff
@@ -36,6 +36,25 @@ export default {
       update(data) {
         return data.undelegations
       }
+    },
+    $subscribe: {
```

Comment:
so we have to add this `$subscribe` for transactions on every component that could change?

Reply:
this is my approach to have each individual component react to a potential update

Reply:
The subscription should then refetch the required queries.